packages/playgrounds/src/sharing.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { useCallback, useState } from 'react'
+import { createStore, combineReducers } from 'redux'
+import { Provider } from 'react-redux'
+import ReactDOM from 'react-dom'
+
+import SharingProvider, { ShareButton, ShareModal } from 'cozy-sharing'
+import withLocales from 'cozy-sharing/dist/withLocales'
+
+import App from './common/App'
+import client from './common/client'
+
+import { Route } from 'react-router'
+
+import 'cozy-ui/transpiled/react/stylesheet.css'
+import 'cozy-ui/dist/cozy-ui.min.css'
+import 'cozy-sharing/dist/stylesheet.css'
```

Comment:
I'm not a big fan of that. Why not doing like cozy-scanner ie having the module importing the stylesheet.css itself where needed? 

https://github.com/cozy/cozy-libs/blob/master/packages/cozy-scanner/src/CategoryGridItem.jsx#L7

https://github.com/cozy/cozy-libs/blob/master/packages/cozy-scanner/babel.config.js

Reply:
Ok, je fais la modif de suite. Si je remets à plus tard ça risque de ne pas être fait.

Reply:
Il faut éliminer stylus pour ça et charger du pur CSS à la place.. C'est tout à fait jouable mais il y a quelques `extends` à résoudre et j'ai peur d'y passer un peu trop de temps alors que ce n'est pas la priorité. Je vais suivre ton commentaire et remettre ça à plus tard.

Reply:
https://github.com/cozy/cozy-libs/issues/887